common/djangoapps/entitlements/tasks.py

```diff
@@ -127,13 +134,17 @@ def expire_and_create_entitlements(self, entitlement_ids, support_username):
                 'support_user': support_user,
             }
             CourseEntitlementSupportDetail.objects.create(**support_detail)
-            LOGGER.info(
-                'created new entitlement with id %d corresponding to above expired entitlement with id %d',
+            log.info(
+                'created new entitlement with id %d corresponding to above expired entitlement with id %d, task id :%s ',
                 new_entitlement.id,
                 entitlement.id,
+                self.id
             )
 
     except Exception as exc:  # pylint: disable=broad-except
-        LOGGER.exception('Failed to expire entitlements that reached their expiration period')
+        log.exception('Failed to expire entitlements that reached their expiration period, task id :%s',
+                      self.id)
 
-    LOGGER.info('Successfully completed the task expire_and_create_entitlements after examining %d entries', len(entitlement_ids))  # lint-amnesty, pylint: disable=line-too-long
+    log.info('Successfully completed the task expire_and_create_entitlements after examining %d entries, task id :%s',
+             len(entitlement_ids),
+             self.id)  # lint-amnesty, pylint: disable=line-too-long
```

Comment:
You may not need the lint-amnesty anymore:

```suggestion
             self.id)
```